core/state_processor_test.go

```diff
@@ -616,3 +620,838 @@ func getParentBlockHash(statedb *state.StateDB, number uint64) common.Hash {
+			if err := tx1.UnmarshalBinary(tx1payload); err != nil {
+				t.Fatal(err)
+			}
+			gen.AddTx(&tx1)
+
+			// SSTORE at slot 133 and reverts
+			tx2payload := common.Hex2Bytes("02f8db83010f2c01843b9aca0084479c2c18830186a08080b88060006085553fad6000600a55600060565555600060b55506600060cf557f1b8b38183e7bd1bdfaa7123c5a4976e54cce0e42049d841411978fd3595e25c66019527f0538943712953cf08900aae40222a40b2d5a4ac8075ad8cf0870e2be307edbb96039527f9f3174ff85024747041ae7a611acffb987c513c088d90ab288aec080a0cd6ac65ce2cb0a912371f6b5a551ba8caffc22ec55ad4d3cb53de41d05eb77b6a02e0dfe8513dfa6ec7bfd7eda6f5c0dac21b39b982436045e128cec46cfd3f960")
+			if err := tx2.UnmarshalBinary(tx2payload); err != nil {
+				t.Fatal(err)
+			}
+			gen.AddTx(&tx2)
+
+			// this one is a simple transfer that succeeds, necessary to get the correct nonce in the other block.
+			tx3payload := common.Hex2Bytes("f8e80184479c2c18830186a094bbbbde4ca27f83fc18aa108170547ff57675936a80b8807ff71f7c15faadb969a76a5f54a81a0117e1e743cb7f24e378eda28442ea4c6eb6604a527fb5409e5718d44e23bfffac926e5ea726067f772772e7e19446acba0c853f62f5606a526020608a536088608b536039608c536004608d5360af608e537f7f7675d9f210e0a61564e6d11e7cd75f5bc9009ac9f6b94a0fc63035441a83021e7ba04a4a172d81ebb02847829b76a387ac09749c8b65668083699abe20c887fb9efca07c5b1a990702ec7b31a5e8e3935cd9a77649f8c25a84131229e24ab61aec6093")
+			if err := tx3.UnmarshalBinary(tx3payload); err != nil {
+				t.Fatal(err)
+			}
+			gen.AddTx(&tx3)
+		} else {
+			var tx types.Transaction
+			// immediately reverts
+			txpayload := common.Hex2Bytes("01f8d683010f2c028443ad7d0e830186a08080b880b00e7fa3c849dce891cce5fae8a4c46cbb313d6aec0c0ffe7863e05fb7b22d4807674c6055527ffbfcb0938f3e18f7937aa8fa95d880afebd5c4cec0d85186095832d03c85cf8a60755260ab60955360cf6096536066609753606e60985360fa609953609e609a53608e609b536024609c5360f6609d536072609e5360a4609fc080a08fc6f7101f292ff1fb0de8ac69c2d320fbb23bfe61cf327173786ea5daee6e37a044c42d91838ef06646294bf4f9835588aee66243b16a66a2da37641fae4c045f")
+			if err := tx.UnmarshalBinary(txpayload); err != nil {
+				t.Fatal(err)
+			}
+			gen.AddTx(&tx)
+		}
+	})
+
+	tx1ContractAddress := crypto.CreateAddress(account1, 0)
+	tx1ContractStem := utils.GetTreeKey(tx1ContractAddress[:], uint256.NewInt(0), 105)
+	tx1ContractStem = tx1ContractStem[:31]
+
+	// Check that values 0x29 and 0x05 are found in the storage (and that they lead
+	// to no update, since the contract creation code reverted)
+	for _, stemStateDiff := range statediffs[0] {
+		// Check that the value 0x85, which is overflowing the account header,
+		// is present.
+		if bytes.Equal(stemStateDiff.Stem[:], common.Hex2Bytes("917f78f74226b0e3755134ce3e3433cac8df5a657f6c9b9a3d0122a3e4beb0")) {
```

Comment:
Can you please derive the stem by code instead of hard coding here?

It should be the slot (133) key stem of the second contract, but the constant value makes it very hard to understand